Release40update

```diff
@@ -0,0 +1,27 @@
+### Highlights
+ 
+- We're excited to announce availability of the Public Preview of our latest Azure API Management pricing tiers: [Basic v2 and Standard v2](https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview).
+- API Management is now Generally Available in the Israel Central and Italy North Azure regions.
+ 
+### New features, fixes and improvements
+ 
+- You can now specify timeout in milliseconds in the the `forward-request` policy via the new `timeout-ms` attribute.
+- We changed `validate-jwt` policy to properly handle `Bearer` and other authorization token schemas correctly in custom headers.
```

Comment:
We should remove this, I deliberately did not share this as  there was a regression =)

Can you please remove this?